### DISCUSSION TITLE
Question about Recipe in blitz.js??

### DISCUSSION THREAD

**Discussion by DataCrusade1999** [blitz-js/blitz#3757] [Q&A] [answered]
I want to use Recipe in blitz 2.0 but it isn't supported yet can anyone suggest me another way to install a recipe in a blitz if there is one, I'm trying to install tailwind.

**Comment by MrLeebo** [comment, marked as answer]
The recipes are fairly self-documenting, you can see the steps that the recipe is performing by looking at the code. https://github.com/blitz-js/legacy-framework/blob/main/recipes/tailwind/index.ts These steps are easy to perform manually:

- Install your dependencies 
```bash
yarn add tailwindcss@3 autoprefixer@10 postcss@8
# or
npm install tailwindcss@3 autoprefixer@10 postcss@8
```
- Copy the files from the templates/ folders into your project:

1. copy `postcss.config.js` and `tailwind.config.js` into your project root
2. copy `styles/index.css` into `app/core/`
3. add the `import 'app/core/styles/index.css';` import statement to your app

**Reply by DataCrusade1999** [reply]
Thanks! it's working now🎉🎉
